web/docker/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ "$(id -u)" = '0' ]; then
+  # Set PostgreSQL password file from secrets.
+  pgpass=/run/secrets/pgpass
```

Comment:
Where does this file/binary/??? come from? It's not documented anywhere (at least in the patch).